.github/workflows/main.yml

```diff
@@ -217,7 +217,7 @@ jobs:
         env:
           CERT_MANAGER: "cert-manager" # enables jetstack/cert-manager integration
           BOOKSTORE_SVC: "bookstore-v1"
-          BOOKTHIEF_EXPECTED_RESPONSE_CODE: "404"
+          BOOKTHIEF_EXPECTED_RESPONSE_CODE: "0"
```

Comment:
why does this become '0' ? Shouldn't it still be 404

Reply:
nope... 404 is returned by RDS right now, as there's no route for traffic that's actually listened to
with this change, there's no filter chain that matches bookthief's traffic destination (because it's not permitted), the stream doesn't even reach RDS in this case.

Reply:
but isn't that incorrect? we should be able to show that if a traffic target isn't applied a service can't access and that should be depicted by a 503 or 404. A 0 doesn't seem right to me

Reply:
Several discussions stirred from this:
- SMI is vague, in which it says traffic is to be "denied", but doesn't specifically say what kind of deny (network deny, L7 deny through http code)
- We might proceed with this at the expense of the behavior change, however we will validate this is in line with expected behavior with customers as it does indeed reduce the visibility as of why a connection is rejected, for example.